services/f-droid/f-droid.service.js

```diff
@@ -61,14 +94,15 @@ module.exports = class FDroid extends BaseService {
       base: 'f-droid/v',
       format: '(.+)',
       capture: ['appId'],
+      queryParams: ['metadata_format'],
     }
   }
 
   static get examples() {
     return [
       {
         title: 'F-Droid',
-        exampleUrl: 'org.thosp.yourlocalweather',
+        exampleUrl: 'org.thosp.yourlocalweather?metadata_format=yml',
         pattern: ':appId',
         staticExample: this.render({ version: '1.0' }),
         keywords: ['fdroid', 'android', 'app'],
```

Comment:
Since `metadata_format` is optional, could you add a second example called something like **F-Droid (explicit metadata format)**, and restore the first one? If you use `query: { metadat_format: 'yml' }` it should render correctly in the frontend.